tools/oesign/oesign.c

```diff
@@ -378,6 +465,29 @@ void _merge_config_file_options(
     if (options->security_version.has_value)
         properties->config.security_version = options->security_version.value;
 
+    if (_is_kss_supported())
```

Comment:
Enclave can be signed on non-SGX machines. Don't check `_is_kss_supported()` here.

Reply:
> The oesign tool can't rely on runtime environment checking to set this bit when setting the KSS support flag since enclaves can be signed on non-SGX machines, not just non-KSS supporting machines. oesign should just eliminate the check for _is_kss_supported. 

This still needs to be addressed.

Reply:
Removed _is_kss_supported from Oesign